routes/confirmation/confirmation.controller.js

```diff
@@ -2,9 +2,26 @@ const { checkSchema } = require('express-validator')
 const { doRedirect, renderWithData, checkErrors, getPreviousRoute } = require('./../../utils')
 const { reviewSchema } = require('./../../schemas')
 const { formatAnswerInfo } = require('./checkAnswersFormat')
+const { outputXML } = require('./xmlFormat')
 
 module.exports = function(app) {
-  app.get('/confirmation', renderWithData('confirmation/confirmation'))
+  app.get('/confirmation', (req, res) => {
+    res.render('confirmation/confirmation', {
+      data: req.session,
+      prevRoute: getPreviousRoute(req),
+      answerInfo: formatAnswerInfo(req),
+    })
+
+    outputXML(req, true)
+  })
+
+  app.get('/download', (req,res) => {
+    /**
+     * I imagine back in outputXML, we'll want to disable the download button until the function is done. We'll rarely see it happen, but just to avoid any issues around downloading and incomplete file
+     */
+    const file = './xml_output/taxfile-2018.xml'
+    res.download(file)
```

Comment:
If the file isn't there, this throws an error. 

Let's do a `try`/`catch` maybe and just `res.status(500).send('Some generic message')`.

Better than reading out the directory structure.

![image](https://user-images.githubusercontent.com/2454380/68704630-9a2c2000-055a-11ea-95e1-74e150655fef.png)